internal/builtinfunctions/functions.go

```diff
@@ -380,7 +389,7 @@ func getToLowerFunction() schema.CallableFunction {
 		schema.NewDisplayValue(
 			schema.PointerTo("toLower"),
 			schema.PointerTo(
-				"Outputs the input with Unicode letters mapped to their lower case.",
+				"Returns a version of the input with Unicode letters mapped to their lower case form.",
```

Comment:
Actually, an interesting technicality occurs to me: does `toLower` change only "letters" or are there other Unicode symbols with "case" changes? Yes, the Go documentation talks about "letters" but almost everywhere else, including the Unicode standard, uses the more general term "characters", which seems safer.